{{cookiecutter.project_name}}/CMakeLists.txt

```diff
@@ -1,18 +1,20 @@
-cmake_minimum_required(VERSION 3.13.4)
+cmake_minimum_required(VERSION 3.16.3)
 
-# Enable C++11
+# Enable C++14
 if(NOT DEFINED CMAKE_CXX_STANDARD)
-  set(CMAKE_CXX_STANDARD 11)
+  set(CMAKE_CXX_STANDARD 14)
 endif()
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_CXX_EXTENSIONS OFF)
 
+set (EP_GIT_PROTOCOL "https")
```

Comment:
Was this required?

Reply:
This is a somewhat older issue than the others. The git protocol stopped working a while ago as I remember. Feel free to double-check, but I had to update this in my custom apps, that's for sure.